data/_schemas/help.yaml

```diff
@@ -0,0 +1,90 @@
+$schema: http://json-schema.org/draft-07/schema#
+
+additionalProperties: false
+required: [key, enabled, title, body]
+properties:
+  key: {type: string}
+  enabled: {type: boolean}
+  hidden: {type: boolean}
+  message: {type: string}
+  versionRange: {type: string}
+  title: {type: string}
+  body: {type: string}
+  buttons:
+    type: array
+    items:
+      # TODO: convert this to "enum" instead of "oneOf"
```

Comment:
Does this TODO need resolving in this PR?

Reply:
… no

I've tested that `oneOf` properly rejects items if they don't match. I believe that `enum` would provide nicer errors, but I couldn't get it to work.

So, no, this doesn't need to be resolved for a merge.